library/build.gradle

```diff
@@ -56,9 +56,11 @@ android {
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'com.android.support:appcompat-v7:27.1.1'
+
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.19.0'
     testImplementation 'com.google.truth:truth:0.41'
+    testImplementation 'org.assertj:assertj-core:1.7.1'
```

Comment:
What advantage does this have over the `truth` library that was already added?

Reply:
Oh, i've never used `truth`, didn't know about it, I can change this I guess. 
Though if `truth` is 0.41, is this not in Beta?